src/apim_checkout.tf

```diff
@@ -3,7 +3,6 @@
 ##############
 
 module "apim_checkout_product" {
-  count  = var.checkout_enabled || var.pagopa_proxy_enabled ? 1 : 0
   source = "git::https://github.com/pagopa/azurerm.git//api_management_product?ref=v1.0.16"
```

Comment:
why remove this one ?

Reply:
checkout will be enabled in all environments, so I started a refactoring to remove all `count checks`

Reply:
reverted this refactoring, I will do it in a dedicated PR